server/routes/record.routes.ts

```diff
@@ -0,0 +1,18 @@
+import { Router } from 'express';
+import { ChunkController } from '../controllers/chunk.controller';
+import {Chunk} from "../models/chuck.model";
+import {RecordController} from "../controllers/record.controller";
+
+const router = Router();
+
+router.get('/getAllRecords/:userId', RecordController.getAllRecordsById);//
+router.get('/getRecord/:id', RecordController.getRecord);//
```

Comment:
Please align it all to REST API. if needed speak with me.
to get a specific record, we call `router.get('/records/:id')`